packages/web-pkg/src/composables/piniaStores/avatars.ts

```diff
@@ -0,0 +1,52 @@
+import { defineStore, storeToRefs } from 'pinia'
+import { ref, unref } from 'vue'
+import { useUserStore } from './user'
+
+export type AvatarMap = Record<string, string>
+
+export const useAvatarsStore = defineStore('avatars', () => {
+  const avatarMap = ref<AvatarMap>({})
+  const userAvatar = ref<string>()
+
+  const userStore = useUserStore()
+  const { user } = storeToRefs(userStore)
+
+  const addAvatar = (userId: string, avatar: string) => {
+    avatar[userId] = avatar
+  }
+
+  const getAvatar = (userId: string) => {
+    return unref(avatarMap)[userId]
+  }
+
+  const removeAvatar = (userId: string) => {
+    unref(avatarMap)[userId] = null
+  }
+
+  const setUserAvatar = (avatar: string) => {
+    userAvatar.value = avatar
+    avatarMap[unref(user).id] = avatar
+  }
+
+  const removeUserAvatar = () => {
+    userAvatar.value = null
+    unref(avatarMap)[unref(user).id] = null
```

Comment:
When writing data we always use `.value`, so you should do this instead for consistency:

```suggestion
    avatarMap.value[unref(user).id] = null
```